Broken groovy scripts after migration 1.7.1=>1.11.1

### Describe the bug

Hi @lbroudoux , as requested the issue concerning the breaking on groovy script:

```
Caused by: groovy.lang.MissingMethodException: No signature of method: java.util.Date.format() is applicable for argument types: (String) values: [yyyy-MM-dd]
```

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Microcks version or git rev

1.11.1

### Install method (`docker-compose`, `helm chart`, `operator`, `docker-desktop extension`,...)

helm chart

### Additional information

Sample of the groovy script:

```
now = new Date()

  def getPlannedDelivery() {
            return [
              'startingOn': now.format("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"),
              'finishingOn': now.format("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
            ]
          }


def getServicesList() {
            return [
              'Expr48': [
                'erpId': '218664',
                'label': 'Express 48h',
                'deliveryCarriers': [
                  [
                    'carrier': getCarrierByCode('CHRF'),
                    'plannedDelivery': getPlannedDelivery()
                  ]
                ]
              ],

```

--- a/webapp/pom.xml
+++ b/webapp/pom.xml
@@ -158,6 +158,11 @@
       <artifactId>groovy-jsr223</artifactId>
       <version>${groovy.version}</version>
     </dependency>
+    <dependency>
+    <groupId>org.apache.groovy</groupId>
+      <artifactId>groovy-datetime</artifactId>
+      <version>${groovy.version}</version>
+    </dependency>
 
     <dependency>
       <groupId>io.grpc</groupId>

